app/queries/servers/thread.ts

```diff
@@ -52,17 +52,23 @@ export const observeThreadById = (database: Database, threadId: string) => {
     );
 };
 
-export const observeUnreadsAndMentionsInTeam = (database: Database, teamId: string) => {
-    return queryThreadsInTeam(database, teamId, true).observeWithColumns(['unread_replies', 'unread_mentions']).pipe(
-        switchMap((threads) => {
-            let unreads = 0;
-            let mentions = 0;
-            threads.forEach((thread) => {
-                unreads += thread.unreadReplies;
-                mentions += thread.unreadMentions;
-            });
-            return of$({unreads, mentions});
-        }),
+export const observeUnreadsAndMentionsInTeam = (database: Database, teamId?: string, includeDmGm?: boolean): Observable<{unreads: number; mentions: number}> => {
+    const observeThreads = () => queryThreads(database, teamId, true, includeDmGm).
+        observeWithColumns(['unread_replies', 'unread_mentions']).
+        pipe(
+            switchMap((threads) => {
+                let unreads = 0;
+                let mentions = 0;
+                threads.forEach((thread) => {
```

Comment:
forEach is a bit slower than for .. of .. in case you see perf bottlenecks 0/5 in changing it now or after we determine this is an actual problem.